query/autocomplete.js

```diff
@@ -30,9 +30,6 @@ function generateQuery( clean ){
   // input text
   vs.var( 'input:name', clean.text );
 
-  // always 10 (not user definable due to caching)
-  vs.var( 'size', 10 );
-
```

Comment:
removed this because it should be part of the defaults. now that defaults work properly this should just work.